### DISCUSSION TITLE
Has any one wdx plugin support crx file on linux ?

### DISCUSSION THREAD

**Discussion by tankb52** [doublecmd/doublecmd#1468] [General]
I found a wdx plugin to show info from *.xpi (Mozilla compatible addons) and *.crx (Chrome compatible addons) files on Totalcmd website , but it only work on Windows.

Has any one wdx plugin support crx file on linux ? I want rename some crx file on Debian.

**Comment by alexx2000** [comment]
https://doublecmd.h1n.ru/viewtopic.php?p=42533

**Comment by alexx2000** [comment]
https://doublecmd.h1n.ru/viewtopic.php?p=42567#p42567
